frontend/src/metabase/query_builder/components/view/ViewHeader/components/QuestionDataSource/SourceQuestionBreadcrumbs.tsx

```diff
@@ -0,0 +1,51 @@
+import { type ReactNode, useMemo } from "react";
```

Comment:
Extracted from `QuestionDataSource.jsx`, converted to TS & turned `Questions.Loader` usage into RTK Query usage.